test/e2e/run_test.go

```diff
@@ -423,6 +423,9 @@ var _ = Describe("Podman run", func() {
 		session.WaitWithDefaultTimeout()
 		Expect(session.OutputToString()).To(Not(BeEmpty()))
 		Expect(session).Should(ExitCleanly())
+		session = podmanTest.Podman([]string{"exec", "maskCtr2", "mkdir", "/sys/fs/cgroup/testcgroup"})
```

Comment:
this won't work with rootless and cgroupfs because the user doesn't get a delegated cgroup.

It would be better to run something like `grep /sys/fs/cgroup /proc/self/mountinfo` and make sure there is `rw` in the mount flags.